configure.ac

```diff
@@ -91,7 +91,7 @@ SHLIB_VERSION_ARG=""
 # Finished checking, handle options.
 
 AC_ARG_ENABLE(experimental,
-	AC_HELP_STRING([--enable-experimental], [enable experimental code]))
+	AS_HELP_STRING([--enable-experimental], [enable experimental code]))
 
```

Comment:
I can't find the direct reference, but `AC_HELP_STRING` is deprecated (and currently an alias to `AS_HELP_STRING`). See for instance https://autotools.io/autoconf/arguments.html
If you google you'll see that X/Gnome have also been switching away.